bbb-graphql-server/bbb_schema.sql

```diff
@@ -2471,13 +2471,24 @@ CREATE UNLOGGED TABLE "pluginDataChannelEntry" (
 	CONSTRAINT "pluginDataChannel_pkey" PRIMARY KEY ("meetingId","pluginName","channelName","entryId", "subChannelName"),
 	FOREIGN KEY ("meetingId", "createdBy") REFERENCES "user"("meetingId","userId") ON DELETE CASCADE
 );
+
+ALTER TABLE "pluginDataChannelEntry"
+ADD COLUMN "isPublic" boolean
+GENERATED ALWAYS AS (
+    (
+        ("toUserIds" IS NULL OR array_length("toUserIds", 1) = 0) AND
+        ("toRoles" IS NULL OR array_length("toRoles", 1) = 0)
+    )
+) STORED;
+
 create index "idx_pluginDataChannelEntry_pk_reverse" on "pluginDataChannelEntry"("pluginName", "meetingId", "channelName", "subChannelName");
 create index "idx_pluginDataChannelEntry_pk_reverse_b" on "pluginDataChannelEntry"("channelName", "pluginName", "meetingId", "subChannelName");
 create index "idx_pluginDataChannelEntry_pk_reverse_c" on "pluginDataChannelEntry"("subChannelName", "channelName", "pluginName", "meetingId");
 create index "idx_pluginDataChannelEntry_channelName" on "pluginDataChannelEntry"("meetingId", "pluginName", "channelName", "toRoles", "toUserIds", "subChannelName", "createdAt") where "deletedAt" is null;
 create index "idx_pluginDataChannelEntry_roles" on "pluginDataChannelEntry"("meetingId", "toRoles", "toUserIds", "createdAt") where "deletedAt" is null;
+create index "idx_pluginDataChannelEntry_createdAt" ON "pluginDataChannelEntry"("createdAt" DESC) where "deletedAt" is null;
```

Comment:
Indexes by `plugin` or `dataChannel` will no longer be used as now the client query all entries and doesn't filter by anything other than `meetingId` and `userId`.
I suggest creating two indexes, onde for each view:

```suggestion
create index "idx_pluginDataChannelEntry_public" ON "pluginDataChannelEntry"("meetingId", "createdAt" DESC NULLS FIRST) where "deletedAt" is null and "isPublic" is true;
create index "idx_pluginDataChannelEntry_private" ON "pluginDataChannelEntry"("meetingId", "createdAt" DESC NULLS FIRST) where "deletedAt" is null and "isPublic" is false;
```